plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/BootLoggingConfiguration.java

```diff
@@ -96,7 +96,7 @@ public class BootLoggingConfiguration { //extends AbstractLogEnabled {
             "user.dir",
             "user.home"
     );
-
+    private static final String KEY_OVERRIDES = "keyOverrides";
```

Comment:
@jamezp , the bump to latest wildfly-core implied a change in the set of existing properties for the json-formatter. I attempted this fix. let me know... Thank-you.

Reply:
Ah, probably because of https://github.com/wildfly/wildfly-core/pull/5492. They weren't correctly updated before.